### PR TITLE
Add cURL to the Bash users list in README.md

### PR DESCRIPTION
**Pull request by vmsilvamolina**
## PR Summary

add the cURL example on the Bash users list

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **User-facing changes**
    - [x] Not Applicable
    - **OR**
    - [ ] User-facing [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed - Issue link:
- **Testing - New and feature**
    - [x] Not Applicable or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)
        - [ ] [Add `[feature]` if the change is significant or affects feature tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/testing-guidelines.md#requesting-additional-tests-for-a-pr)

### CODE REVIEW

File: docs/learning-powershell/README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Note that all bash commands should continue working on PowerShell session.
 | rm -r &lt;folderName>           |Remove-Item &lt;folderName> -Recurse   |Delete a folder
 | find -name build*               |Get-ChildItem build* -Recurse          |Find a file or folder starting with 'build'
 | grep -Rin "sometext" --include="*.cs" |Get-ChildItem -Recurse -Filter *.cs <br> \| Select-String -Pattern "sometext" | Recursively case-insensitive search for text in files
-
+| curl dominio.com | Invoke-WebRequest dominio.com | Transfer data from or to a web from differentes protocols
```
Comment (iSazonov) [inline docs/learning-powershell/README.md]: Maybe `Transfer data from web`?

Comment (SteveL-MSFT) [reply docs/learning-powershell/README.md]: I think `Invoke-RestMethod` is more similar to `curl` than `Invoke-WebRequest`.  Also, the example does work with curl and it's better to use a more generic URL:

> curl https://github.com
> Invoke-RestMethod https://github.com

For the description, maybe: Transfer data to or from the web

**Review by SteveL-MSFT — CHANGES_REQUESTED** [review]

File: docs/learning-powershell/README.md (hunk 1 above)
Comment (vmsilvamolina) [reply docs/learning-powershell/README.md]: Ok, I'll make the changes.

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Review by TravisEz13 — APPROVED** [review]

**Comment by iSazonov** [comment]
@vmsilvamolina Thanks for your contribution!

**Pull request merged**